website/docs/deployment.mdx

```diff
@@ -397,3 +397,25 @@ echo subdomain.surge.sh > CNAME
 ```
 
 You can deploy any other changes in the future with the command `surge`.
+
+## Deploying to QuantCDN
+
+## QuantCDN
+
+1. Install [Quant CLI](https://docs.quantcdn.io/docs/cli/get-started)
+
+2. Create a QuantCDN account by [singing up](https://dashboard.quantcdn.io/register)
```

Comment:
I think you have a typo here. 😉 
Should probably be: `signing up`.

Reply:
Singing gives you a secret bonus 😛 , thanks for picking that up!